src/main/webapp/app/exercises/modeling/assess/modeling-assessment-editor/modeling-assessment-dashboard.component.ts

```diff
@@ -261,5 +263,18 @@ export class ModelingAssessmentDashboardComponent implements OnInit, OnDestroy {
         this.eventManager.destroy(this.eventSubscriber);
     }
 
-    callback() {}
+    public sortRows() {
+        if (this.predicate === 'durationInMinutes') {
```

Comment:
Same comment as for file upload exercises

Reply:
Addressed via: 55d5b75